specs/Separate_conf_files.adoc

```diff
@@ -0,0 +1,141 @@
+= Separate out per-component configuration files
+
+Configuration details for the tendrl modules are read from same file(tendrl.conf).
+Based on the tendrl modules we have to keep the configuration details into different
+files with proper name and read it independently.
+
+== Problem description
+
+Currently all the configuration details for different modules are read from same file .
+We have to split the configuration details as per modules and keep that into different
+file with proper name. Each configuration file should have its own and relevant configuration
+information only.
+
+We have to modify the python file (config.py) which read configuration file as per this
+modification. The python file should read each configuration files and it has to combine all
+the information into single object.
+
+== Use Cases
+
+None
+
+== Proposed change
+
+** Modify the sample configuration files in each modules. It should contains its own
```

Comment:
i have added some sub bullets points under this bullet points, So it will separate bullet points from sub bullet points